src/Microsoft.Identity.Client/PlatformsCommon/Shared/AbstractPlatformProxy.cs

```diff
@@ -44,6 +44,14 @@ public virtual bool IsSystemWebViewAvailable
             }
         }
 
+        public virtual bool UseEmbeddedWebViewDefault
+        {
+            get
+            {
+                return true;
```

Comment:
@jennyf19  - could you  please review this Commit? I made the defaults explicit in PlatformProxy and wanted to get your opinion if I broken anything. embeddedWebview is set to false on iOS and Android right?

Reply:
yes...default is system browser on iOS and Adnroid

---
In reply to: [285590139](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1160#discussion_r285590139) [](ancestors = 285590139)